tests/test_events.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+from celery import Celery
+from flask import Flask
+from flask.json import jsonify, dumps
+from flask.ext.notifications import NotificationService, Event
```

Comment:
Please use `from flask_notifications import ...`. `flask.ext` import hook is going to be deprecated in Flask 1.0. Thanks